ios/FluentUI.Demo/FluentUI.Demo/Demos/ListDemoController.swift

```diff
@@ -27,36 +27,23 @@ class ListDemoController: DemoController {
             PersonaData(name: "Amanda Brady", subtitle: "Program Manager", image: UIImage(named: "avatar_amanda_brady"), color: Colors.Palette.magentaPink10.color)
         ]
 
-        var avatar: MSFAvatar
-
         /// Custom Leading View with collapsible children items
         let collapsibleSection = list.state.createSection()
         collapsibleSection.title = "AvatarSection"
-        for index in 0...1 {
+        for samplePersonaIndex in 0...1 {
```

Comment:
Do you think you can create a struct with the sample data in a hierarchical organization and replicate the creation of these cells based on it? It will avoid the multiple for statements in these odd ranges (0...1, 2...3)...

Something like:

```
struct PersonaDataNode {
  let personaData: PersonaData
  let children: [PersonaData]
  let isCollapsed: Bool
 ....
}
```